vj4/model/builtin.py

```diff
@@ -60,13 +60,22 @@
 
 # Contest.
 PERM_VIEW_CONTEST = 1 << 41
-PERM_VIEW_CONTEST_STATUS = 1 << 42
-PERM_VIEW_CONTEST_HIDDEN_STATUS = 1 << 43
+PERM_VIEW_CONTEST_SCOREBOARD = 1 << 42
```

Comment:
Code name is `STATUS`

Reply:
STATUS is renamed to SCOREBOARD. Currently we have too many "status", referring to different things, which is very confusing. For example, when talking about a contest's status, you may refer to "ongoing / finished", or may refer to "scoreboard".